proto/migration/v1alpha1_to_v1.go

```diff
@@ -366,6 +367,140 @@ func V1Alpha1BeaconBlockAltairToV2(v1alpha1Block *ethpbalpha.BeaconBlockAltair)
 	return v2Block, nil
 }
 
+func BeaconStateToV1(state *statev1.BeaconState) (*ethpbv1.BeaconState, error) {
```

Comment:
Could you add a comment explaining what this method is for? I found the naming to be somewhat confusing. `migration` to me is more like a mutation operation than a transform operation. "Change the underlying thing to fit a new schema" vs "make a copy of the thing, but transform it to fit a different co-existing schema". And both the input and the output are in packages with `v1` in the name, so the `ToV1` aspect of the name doesn't say much. I think the spirit of this method is that you want to transform between our internal BeaconState representation and the proto representation.